.gitignore

```diff
@@ -6,7 +6,6 @@
 /deployment/public/img
 /deployment/public/assets
 /deployment/public/Neon
-/deployment/public/uploads/*
```

Comment:
Why the changes to the gitignores? This doesn't seem to be related to the linked issue.

Reply:
It was for  development purposes, we got multiple pages running in the `yarn build` by importing `mei` and `manifests` folders from the `gh-pages` branch. It was easier to deal with the imports using local `.gitignore` files rather than ignoring the entire `uploads` directory.